src/app/components/icons/IconDrawer.tsx

```diff
@@ -69,6 +80,11 @@ export const IconDrawer: React.FC = () => {
     const history = useHistory();
     const dispatch = useDispatch();
     const classes = useStyles(theme);
+    type IconSize = 18 | 24 | 36 | 48;
+    type IconColor = 'Black' | 'Blue' | 'Gray' | 'White';
```

Comment:
I would suggest defining these in the shared types fie (https://github.com/pxblue/doc-it/blob/dev/src/__types__/index.ts) and export them so we can use them in the utility functions.